### ISSUE TITLE
Resolve priority 

### ISSUE THREAD

**Issue by godcore** [lpiepiora/bower-webpack-plugin#20]
Hi!I found that this plugin ignores

**Comment by mathieumg** [comment]
I have the same issue! :+1: See my message in https://github.com/webpack/enhanced-resolve/pull/15#issuecomment-113864959

**Comment by godcore** [comment]
Yes, thnx, I've already found your solution and have applied it to my build setup. It works perfectly. However PR is not accepted yet. And I also believe ```bower-webpack-plugin``` should have options to handle such cases

**Comment by mathieumg** [comment]
> And I also believe bower-webpack-plugin should have options to handle such cases

Yep, right now the problem (for this plugin) is not that it ignores `resolve.modulesDirectories` (see https://github.com/lpiepiora/bower-webpack-plugin/blob/master/lib/bower-plugin.js#L63) but rather that it specializes in finding Bower modules exclusively, so it would overlook any module that doesn't have a `bower.json` (by default, but the value of the `manifestFiles` configuration otherwise) file in it. (See https://github.com/lpiepiora/bower-webpack-plugin/blob/master/lib/bower-plugin-manifest-resolver.js#L107)

Thus, that is why I like the solution in the `enhanced-resolve` PR better since it's tied more closely to the Webpack pipeline and lookup strategy.

**Comment by Ridermansb** [comment]
Hi!.

Sorry for my ignorance, but I dont understand how to fix this.

In my case I want to use [stamplay-js-sdk](https://github.com/Stamplay/stamplay-js-sdk) only available in Browser but I cant.
I would just load this library (and their dependents `store` and `Q`) of bower, the rest continue require from NPM

I created a [sample app](https://github.com/Ridermansb/webpackBowerStarter) to tests this and [post this question](http://stackoverflow.com/questions/33502112/webpack-with-bower-support) on stackoverflow too

Error when run `./node_modules/.bin/webpack`

```
ERROR in ./app.js
Module not found: Error: Cannot resolve module 'stamplay-js-sdk' in {{OMITED}}\webpackBowerStarter
 @ ./app.js 7:15-41
```

Any help will be appreciated. :)
